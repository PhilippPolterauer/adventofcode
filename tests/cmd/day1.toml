bin.name = "adventofcode"
args = ["1", "1"]

[env.add]
write = "file.txt = Goodbye\r\nWorld\r\n"
cat = "file.txt"
stdout = "Hello\r\nWorld!\r\n"
