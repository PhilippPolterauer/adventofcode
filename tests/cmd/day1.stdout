day: 1, part: 1
loading data from 'data/day1/input1.txt'
Solution: 55447
